Makefile

```diff
@@ -1,12 +1,22 @@
 all: test
 
-uninstall:
-	pip freeze | grep -v "^-e" | xargs pip uninstall -y
-
-install:
+install.tools:
 	pip install --upgrade pip twine wheel
+
+## Install openfisca-core for deployment and publishing.
+install.build: setup.py clean install.tools
+	@## This allows us to be sure tests are run against the packaged version
+	@## of openfisca-core, the same we put in the hands of users and reusers.
+	pip install --upgrade pip wheel
+	python $< bdist_wheel
+	find dist -name "*.whl" -exec pip install --force-reinstall {}[dev] \;
+
+install: install.tools
```

Comment:
I had initially understood `install` as being for “usage” only, but as it depends on `install.tools`, I understand that running `make install` will also install [`twine`](https://pypi.org/project/twine/), which is “a utility for publishing Python packages on PyPI”, which seems appropriate for the `install.build` target considering its documentation, not for `install`.

Reply:
Semantically does not make a lot of sense, thanks for the feedback.

Reply:
Fixed.